docs/source/overview.rst

```diff
@@ -4,17 +4,60 @@ Overview of Iroha
+
 Where Iroha can be used?
 ------------------------
 
+Hyperledger Iroha is a general purpose permissioned blockchain system that can be used to manage digital assets, identity, and serialized data.
+This can be useful for applications such as interbank settlement, central bank digital currencies, payment systems, national IDs, and logistics, among other.
+
+For a detailed description please check our `Use Case Scenarios section <http://iroha.readthedocs.io/en/latest/use_cases/>`_.
+
 How is it different from Bitcoin or Ethereum?
 ---------------------------------------------
 
+Bitcoin and Ethereum are designed to be permissionless ledgers where anyone can join and access all the data.
+They also have native cryptocurrencies that are required to interact with the systems.
+
+In Iroha there is no native cryptocurrency. Instead, to meet the needs of enterprises, system interaction is permissioned, meaning that only people with requisite access can interact with the system. Additionally, queries are also permissioned, such that access to all the data can be controlled.
+
+
+One major difference from Ethereum, in particular, is that Hyperledger Iroha allows users to perform common functions, such as creating and transferring digital assets, by using prebuilt commands that are in the system.
+This negates the need to write cumbersome and hard to test smart contracts, enabling developers to complete simple tasks faster and with less risk.
+
 How is it different from the rest of Hyperledger frameworks or other permissioned blockchains?
 ----------------------------------------------------------------------------------------------
 
+Iroha has a novel, Byzantine fault tolerant consensus algorithm (called YAC [#f1]_) that is high-performance and allows for finality of transactions with low latency.
+Other frameworks either focus more on probabilistic consensus algorithms, such as Nakamoto Consensus, or sacrifice Byzantine fault tolerance.
+
+Also, Iroha's built-in commands are a major benefit compared to other platforms, since it is very simple to do common tasks such as create digital assets, register accounts, and transfer assets between accounts.
+
+Finally, Iroha is the only ledger that has a robust permission system, allowing permissions to be set for all commands, queries, and joining of the network.
+
+.. [#f1] Yet Another Consensus
+
 Is it fast?
 -----------
 
+As per the latest review date of these docs, according to `Huawei Caliper <https://github.com/hyperledger/caliper>`_ testing tool, Iroha is capable of processing 45 transactions per second. Theoretically, this is not even close to the limit of the system, and we will continue constant optimizations in order to improve stability and performance.
+
 How to create applications around Iroha?
 ----------------------------------------
+
+In order to bring power of blockchain into your application you should think first of how it is going to interface with Iroha peers.
```

Comment:
Probably **the** article is missing: In order to bring **the** power of blockchain ...

Reply:
Please consider adding a comma after "application":
In order to bring ... into your application, you should think ...